staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,24 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container to manage the restart behavior of each
+	// container within a pod.
+	// By default, if no value is assigned to this field, the restart behavior of
+	// the container follows the Pod's RestartPolicy. This field may only be set
+	// for init containers and only allowed value is "Always".
+	// "Always" results in this init container being continually restarted on
+	// exit until all regular containers have terminated. Once all regular
+	// containers have terminated and not being restarted, the init container
```

Comment:
`... have completed, all init containers with restartPolicy "Always" will be shut down. ` (we use 'completed' in other places to indicate the pod will not be restarted and is easier to read).